setup.py

```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 
-from pathlib import Path
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
```

Comment:
This would only affect Python < 3.4, but we really just support Python >= 3.5.

Does this actually make Python 3.4 work?